tools/rcx-tk/rcx-tk.xml

```diff
@@ -0,0 +1,70 @@
+<tool id="rcx-tk" name="rcx-tk" version="0.1.0+galaxy0" python_template_version="3.5" profile="23.05">
+    <edam_topics>
+        <edam_topic>topic_3172</edam_topic>
+        <edam_topic>topic_0091</edam_topic>
+        <edam_topic>topic_3520</edam_topic>       
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_0336</edam_operation>
+        <edam_operation>operation_3695</edam_operation> 
+    </edam_operations>
+    <requirements>
+        <requirement type="package" version="0.1.0">rcx-tk</requirement>
+    </requirements>
+    <creator>
+        <person
+                givenName="Zargham"
+                familyName="Ahmad"
+                url="https://github.com/zargham-ahmad"
+                identifier="0000-0000-0000-0000"/>
+        <person
+                givenName="Kristina"
+                familyName="Gomoryova"
+                url="https://github.com/KristinaGomoryova"
+                identifier="0000-0003-4407-3917" />
+        <person
+                givenName="Helge"
+                familyName="Hecht"
+                url="https://github.com/hechth"
+                identifier="0000-0001-6744-996X" />
```

Comment:
Please also add RECETOX as an organization :)

Reply:
resolved in cab382ef227760397b9ea570fea5583a3c00dd42